analyticskit/build.gradle

```diff
@@ -55,6 +56,27 @@ dependencies {
 	testImplementation "org.jetbrains.kotlin:kotlin-test-junit:$rootProject.ext.kotlinVersion"
 }
 
+afterEvaluate {
+	publishing {
+		publications {
+			releaseAnalyticsKit(MavenPublication) { // Creates a Maven publication called "releaseAnalyticsKit".
+				from components.release // Applies the component for the release build variant.
+
+				groupId = 'com.github.busybusy.AnalyticsKit-Android'
+				artifactId = 'analyticskit'
+				version = '0.9.2'
```

Comment:
I was just thinking about this - the version number is repeated in each module's build file. Since we release all artifacts at the same time under the same version number, the version number would be a good candidate to move to `gradle.properties`.